Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -102,7 +104,10 @@ extension PhotonActionSheetProtocol {
             vcDelegate.present(controller, animated: true, completion: nil)
         }
 
-        return [noImageMode, nightMode, openSettings]
+        if let noImageMode = noImageMode {
+            return [noImageMode, nightMode, openSettings]
+        }
+        return [nightMode, openSettings]
```

Comment:
I think in the general case, where we differentially include options, I'd prefer to have a list of `options` which we append.

```swift
var options = [PhotonActionSheetItem]()

let nightModeItem = …
options.append(nightModeItem)

if #available(iOS11, *) {
    let noImageModeItem = …
    options.append(noImageModeItem)
}

return options
```

For this simple case, ok.

Reply:
thanks, will use way that in future